### PR TITLE
added scripts to ideal gas tutorial with options to initialize with r…

### PR DESCRIPTION
**Pull request by rosecers**
added scripts to ideal gas tutorial with options to initialize with replica indices or add replica indices to existing jobs

#1 https://github.com/glotzerlab/signac-docs/issues/1

### CODE REVIEW

File: examples/tutorial/add_rep_indices.py [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+jobs = project.find_jobs({"replica_index.$exists":False})
+for job in jobs:
+    job.sp['replica_index']=0
+
+for i in range(num_reps) :
+    for p in range(1, 11):
+        sp = {'p': p, 'kT': 1.0, 'N': 1000, "replica_index": i}
+        job = project.open_job(sp)
+        if(job not in project):
+            job.init()
```
Comment (csadorf) [inline examples/tutorial/add_rep_indices.py]: It's safe to call `init()` on an existant job, so you could just call `init()` in line 14 and remove lines 15 and 16.

**Comment by vyasr** [comment]
@csadorf is this stalled for any particular reason? I agree that the recipes are a good place to put this, if @yuanzhou0827 can move this code there.

**Comment by csadorf** [comment]
> @csadorf is this stalled for any particular reason?

No, I just forgot about it.

**Comment by vyasr** [comment]
@yuanzhou0827 would you be able to move this code to the recipes section of the docs? Then it would always show up in the documentation.

**Comment by yuanzhou0827** [comment]
Since it was not edited in my repo, can I pull a new request of adding the codes in Signac-docs recipe.rst file instead?

**Comment by vyasr** [comment]
Yes! That's a good question. Since the original changes were made in Rosy's fork, you'll need to add her repository as a remote to your signac-docs repository so that you can pull from it. If you want to learn more about how to do that, you can look at the [git book](https://git-scm.com/book/en/v2/Git-Basics-Working-with-Remotes), but for you it should be enough to just do this:
```
git remote add rosecers https://github.com/rosecers/signac-docs
git fetch rosecers
git checkout --track rosecers/master -b replica_recipes
```
You'll then have a new branch `replica_recipes` that will contain the changes currently on Rosy's branch. You will probably need to make a new pull request, and we can then close this one.

**Comment by mikemhenry** [comment]
Let me know if you need any help @yuanzhou0827 in sorting out the git remotes!

**Comment by vyasr** [comment]
Superseded by #57

**Pull request closed without merging**